configs/default/FLWO-FLYWOOF411EVO_HD.config

```diff
@@ -0,0 +1,115 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
+
+board_name FLYWOOF411EVO_HD
+manufacturer_id FLWO
+
+# resources
+resource BEEPER 1 C14
+resource MOTOR 1 A08
+resource MOTOR 2 A09
+resource MOTOR 3 A10
+resource MOTOR 4 B10
+resource PPM 1 A02
+resource PWM 1 B01
+resource LED_STRIP 1 B05
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ADC_BATT 1 A00
+resource ADC_RSSI 1 B01
+resource ADC_CURR 1 A01
+resource PINIO 1 B05
+resource FLASH_CS 1 B02
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 B03
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C15
+
+# timer
+timer A02 AF3
+# pin A02: TIM9 CH1 (AF3)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A08 1
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin A09 1
+# pin A09: DMA2 Stream 2 Channel 6
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+
+# feature
+feature SOFTSERIAL
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+feature ANTI_GRAVITY
+feature DYNAMIC_FILTER
+
+# serial
+serial 20 1 115200 57600 0 115200
+serial 0 64 115200 57600 0 115200
+serial 1 8192 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```

Comment:
Will this target only be used for an all-in-one board with on-board ESCs that support Dshot600?